app/views/content/shared/quotes/_quote_engineers_teach_physics_chris_physics.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="quote-chris-physics">
```

Comment:
@joegibb very small point - worth shortening the name of this file to delete the last `physics`, so it is consistent with the other quote?

eg `_quote_engineers_teach_physics_chris.html.erb`